### PR TITLE
PR feat/db-migration into Dev

### PR DESCRIPTION
**Pull request by lpendem**

### CODE REVIEW

File: models/files.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var files = sequelize.define('files', {
```
Comment (cameronwolf) [inline models/files.js]: Should be using let or const

File: config/config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const dotenv = require('dotenv').config();
+
+module.exports = {
+  development: {
```
Comment (cameronwolf) [inline config/config.js]: Indentation using spaces, should be tabs.

Comment (cameronwolf) [inline config/config.js]: Running eslint --fix should clean up these issues.

File: models/files.js (hunk 1 above)
Comment (lpendem) [inline models/files.js]: @cameronwolf this is sequelize code setup, will have to check if that breaks anything.

File: migrations/01-create-filetypes.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+/* eslint-disable camelcase */
+'use strict';
+module.exports = {
+	up: function(queryInterface, Sequelize) {
+		return queryInterface.createTable('file_types', {
+			file_type: {
+				allowNull: false,
+				primaryKey: true,
+				type: Sequelize.STRING,
+				allowNull: false
```
Comment (cameronwolf) [inline migrations/01-create-filetypes.js]: @lpendem Should this key be declared twice?

**Pull request merged**